README.md

```diff
@@ -81,20 +86,19 @@ Welcome to the code for Giveth's dapp. This is an open source effort to realize
    ```
 10. That is it, you are now ready to run the giveth-dapp! Head to the [Run dapp](#run-dapp) section for further instructions.
 
-### Run dapp
-1. The Giveth dapp will need to connect to a [feathers-giveth](https://github.com/Giveth/feathers-giveth) server. Follow the feathers-giveth readme instructions to install and run server before proceeding further.
+### Run
+1. The Giveth dapp will need to connect to a [feathers-giveth](https://github.com/Giveth/feathers-giveth) server. Follow the feathers-giveth readme instructions to install and run server before proceeding further. Alternatively, you could change the configuration to connect to one of our environments, see the [Configuration](#configuration) section.
```

Comment:
this is tricky. we don't want people connecting to `alpha`

Reply:
maybe it's possible to put a restriction/config value in feathers to only accept modification requests (!GET) from a specific domain

Reply:
I think we want people to connect to alpha in the end. It is a great way if you don't trust our UI. Plus its what dappnote is trying to do. But I do see your point.